src/query_plan.c

```diff
@@ -11,9 +11,9 @@
 static size_t serializeResult(QueryPlan *qex, SearchResult *r, RSSearchFlags flags,
                               RedisModuleCtx *ctx) {
   size_t count = 0;
-  if (r->md && !(qex->opts.flags & Search_AggregationQuery)) {
+  if (r->scorerPrivateData && !(qex->opts.flags & Search_AggregationQuery)) {
```

Comment:
The idea was to remove this stuff :) - this is obviosuly not the scorer using this.